rego/input.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I'm not sure how this relates to the input values provided in the example from the README.

Reply:
Just the example input, anw, I will remove it.